crawlerr/services/rag_service.py

```
@@ -28,8 +29,52 @@ class RagService:
+                self.chunk_overlap = 200  # Overlap between chunks
+        
+        # Initialize Qwen3 reranker
+        self.reranker = None
+        self.reranker_type = "none"
+        
+        if settings.reranker_enabled:
+            try:
+                from sentence_transformers import CrossEncoder
+                self.reranker = CrossEncoder(settings.reranker_model)
+                self.reranker_type = "qwen3"
+                logger.info(f"Using Qwen3 reranker: {settings.reranker_model}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)